packages/affiliates/payouts/devmining-status.json

```
@@ -0,0 +1,11 @@
+{
+  "empWhitelist": [
+    "0x3a93E863cb3adc5910E6cea4d51f132E8666654F",
+    "0x516f595978D87B67401DaB7AfD8555c3d28a3Af4",
+    "0x4AA79c00240a2094Ff3fa6CF7c67f521f32D84a2",
+    "0xf32219331A03D99C98Adf96D43cc312353003531",
+    "0x1c3f1A342c8D9591D9759220d114C685FD1cF6b8",
+    "0xE4256C47a3b27a969F25de8BEf44eCA5F2552bD5"
```

Since we expect to support these three ZeldaCash contracts, think we should just add them now so that people can get more accurate rewards predictions. Wdyt?

```suggestion
    "0xE4256C47a3b27a969F25de8BEf44eCA5F2552bD5",
    "0xeAddB6AD65dcA45aC3bB32f88324897270DA0387",
    "0xf215778F3a5e7Ab6A832e71d87267Dd9a9aB0037",
    "0x267D46e71764ABaa5a0dD45260f95D9c8d5b8195"
```

---

yea sounds good